docs/pages/web/radiogroup.js

```diff
@@ -288,6 +289,18 @@ export default function DocsPage({
             }
           />
         </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Adding a Badge"
+          description={`
+    The \`badge\` prop can be used to add a badge to the RadioButton.
+  `}
+        >
+          <MainSection.Card
+            sandpackExample={
+              <SandpackExample name="Adding a popover example" code={addingABadgeExample} />
```

Comment:
Can we rename this to just code={badge}.   ??

Reply:
name="With Badge"

Reply:
Done